tests/comparisons/Model/testGetBehaviors.php

```diff
@@ -0,0 +1,55 @@
+<?php
+namespace App\Model\Table;
+
+use Cake\ORM\Query;
+use Cake\ORM\RulesChecker;
+use Cake\ORM\Table;
+use Cake\Validation\Validator;
+
+/**
+ * Posts Model
+ *
+ * @method \App\Model\Entity\Post get($primaryKey, $options = [])
+ * @method \App\Model\Entity\Post newEntity($data = null, array $options = [])
+ * @method \App\Model\Entity\Post[] newEntities(array $data, array $options = [])
+ * @method \App\Model\Entity\Post|bool save(\Cake\Datasource\EntityInterface $entity, $options = [])
+ * @method \App\Model\Entity\Post patchEntity(\Cake\Datasource\EntityInterface $entity, array $data, array $options = [])
+ * @method \App\Model\Entity\Post[] patchEntities($entities, array $data, array $options = [])
+ * @method \App\Model\Entity\Post findOrCreate($search, callable $callback = null, $options = [])
+ *
+ * @mixin \Cake\ORM\Behavior\CounterCacheBehavior
+ * @mixin \Cake\ORM\Behavior\TranslateBehavior
+ */
+class PostsTable extends Table
+{
+
```

Comment:
We should remove that newline if possible (as per psr2). Or the generated code will always have to be sniffer fixed afterwards.

Reply:
Our coding standard does not report any error for this new line.

Reply:
I checked, neither does PSR2.

Reply:
But it should, it states that there is no newline at beginning or end.